simpledotorg/simple-server
# changing numerator in bs < 200 and over 200 charts to include only fa…

…sting and hba1c measurements

**Story card:** https://app.shortcut.com/simpledotorg/story/13103/rename-blood-sugar-200-299-or-300-to-blood-sugar-uncontrolled
https://app.shortcut.com/simpledotorg/story/13102/rename-blood-sugar-200-card
https://app.shortcut.com/simpledotorg/story/13104/update-blood-sugar-controlled-card

## Because

Sri Lanka wanted to change the numerator in diabetes controlled and uncontrolled chart to include only fasting and hba1c measurements, unlike the current implementation where it includes random measurements as well.

## This addresses

Changes are put behind a feature flag "use_who_standard_for_diabetes_indicator". If this is disabled the indicators would behave as before. 
All the text changes for the dashboard, mentioned in the stories, are locale based and would be effective when DEFAULT_PREFERRED_DASHBOARD_LOCALE is set to "en-LK".
Separate methods have been created to be used based to on the feature flag to only include fasting and hba1c readings for the numerator.

## Test instructions

Enable use_who_standard_for_diabetes_indicator
Set DEFAULT_PREFERRED_DASHBOARD_LOCALE to "en-LK"
Check the diabetes controlled and uncontrolled charts in Simple dashboard, it should now include only fasting and hba1c readings.
